_15thnight/core.py

```diff
@@ -95,10 +96,26 @@ def respond_to_alert(provider, needs_provided, alert):
     return response
 
 
-def send_out_resolution(need):
-    """
-    Sends out a resolution for a need to providers.
-    """
+def send_alert_resolution_notice(need):
```

Comment:
I feel like the resolve_need method is starting to get too large, hence the code breakup. This code breakup also allows for easier testing and an easier path for a possible future "resend final alert closed notification" if needed.